rocAL/rocAL_pybind/amd/rocAL/pipeline.py

```diff
@@ -240,6 +214,22 @@ def copyImage(self, array):
         b.raliCopyToOutput(
             self._handle, np.ascontiguousarray(out, dtype=array.dtype))
 
+    def copyToHipTensorNHWC(self, multiplier, offset, reverse_channels, tensor_dtype):
```

Comment:
better to rename to copyToGpu..() instead of using Hip

Reply:
Renamed